actiontext/lib/rails/generators/actiontext/actiontext_generator.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "pathname"
+require "json"
+
+module Rails
+  module Generators
+    class ActiontextGenerator < Base
+      JS_PACKAGE_PATH = Pathname.new("#{__dir__}/../../../../package.json")
+
+      JS_PACKAGE = JSON.load(JS_PACKAGE_PATH)
```

Comment:
Can we move this to a method? I don't think we should be reading a JSON when this file is loaded.